### PR TITLE
Add Nexus repos

### LINKED ISSUE #76: Deploy artifacts on Nexus
In order to other project to consume quarkus-ls, it would be good to have the artifacts (mainly the LSP uber jar) deployed on Nexus

### PR DESCRIPTION
**Pull request by jeffmaury**
This allows deploying the quarkus ls to Nexus.
I have a Jenkins Job (quarkus-ls) that takes this branch and successfully deployed to Nexus.
As soon as this is merged, I will update so that it works from master.

Fixes #76

Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### CODE REVIEW

**Comment by jeffmaury** [comment]
See https://repository.jboss.org/com/redhat/quarkus/com.redhat.quarkus.ls/

File: quarkus.ls/com.redhat.quarkus.ls/pom.xml [hunk 1]
```diff
@@ -103,4 +107,24 @@
 			<scope>test</scope>
 		</dependency>
 	</dependencies>
+	
+	<pluginRepositories>
+		<pluginRepository>
+			<id>tycho-snapshots</id>
```
Comment (fbricon) [inline quarkus.ls/com.redhat.quarkus.ls/pom.xml]: why is this repo necessary?

File: quarkus.ls/com.redhat.quarkus.ls/pom.xml [hunk 2]
```diff
@@ -103,4 +107,24 @@
...
+			<url>https://repo.eclipse.org/content/repositories/tycho-snapshots/</url>
```
Comment (fbricon) [inline quarkus.ls/com.redhat.quarkus.ls/pom.xml]: should have snapshots enabled, releases disabled

File: quarkus.ls/com.redhat.quarkus.ls/pom.xml (hunk 1 above)
Comment (jeffmaury) [reply quarkus.ls/com.redhat.quarkus.ls/pom.xml]: Bad copy paste, removed it

**Pull request merged**